### DISCUSSION TITLE
I request for Steve.

### DISCUSSION THREAD

**Discussion by steve-is-me** [FunkinCrew/Funkin#1649] [Ideas]

**Comment by ghost** [comment]
Please make sense.

**Comment by ghost** [comment]
happy 1000th issue

**Comment by ghost** [comment]
This may be issue #1000 but it shouldn't exist

**Comment by isakube** [comment]
happy 1000th ninjamuffin this is what you get for being a troll about leaks lol

**Comment by Windows1Door2Daniel** [comment]
Thats trash
